backend/src/main/java/com/woowacourse/levellog/feedback/domain/FeedbackQueryRepository.java

```diff
@@ -0,0 +1,85 @@
+package com.woowacourse.levellog.feedback.domain;
+
+import com.woowacourse.levellog.feedback.dto.FeedbackContentDto;
+import com.woowacourse.levellog.feedback.dto.FeedbackDto;
+import com.woowacourse.levellog.feedback.dto.FeedbacksDto;
+import com.woowacourse.levellog.member.dto.MemberDto;
+import java.util.List;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class FeedbackQueryRepository {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+    private final RowMapper<FeedbackDto> feedbackRowMapper = (resultSet, rowNum) -> new FeedbackDto(
+            resultSet.getObject("feedbackId", Long.class),
+            new MemberDto(
+                    resultSet.getObject("fromId", Long.class),
+                    resultSet.getString("fromNickname"),
+                    resultSet.getString("fromProfileUrl")
+            ),
+            new MemberDto(
+                    resultSet.getObject("toId", Long.class),
+                    resultSet.getString("toNickname"),
+                    resultSet.getString("toProfileUrl")
+            ),
+            new FeedbackContentDto(
+                    resultSet.getString("study"),
+                    resultSet.getString("speak"),
+                    resultSet.getString("etc")
+            ),
+            resultSet.getTimestamp("updatedAt").toLocalDateTime()
+    );
+
+    public FeedbackQueryRepository(final NamedParameterJdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public FeedbacksDto findAllByLevellog(final Long levellogId) {
```

Comment:
```suggestion
    public FeedbacksDto findAllByLevellogId(final Long levellogId) {
```
아주 사소하지만 `findAllByLevellogId(levellogId)`는 어떨까요? 레벨로그 객체가아닌 id를 인자로 보내주고 있고 `JpaRepository` 메서드 네이밍 형식에 따라 일관성있게 맞추는게 좋을거 같아서용!😊